.github/workflows/coordinator-build-and-publish.yml

```diff
@@ -31,6 +31,7 @@ jobs:
   build-and-publish:
     runs-on: ubuntu-22.04
     name: Coordinator build
+    environment: docker-build-and-e2e
```

Comment:
What is `docker-build-and-e2e` environment?

Reply:
This is github environment we've set up which the build and e2e test jobs have been added to. This stops the automatic running of these jobs unless we approve it.